symbiosis.lic

```diff
@@ -0,0 +1,52 @@
+
+custom_require.call(%w[common])
+
+class Symbiosis
+
+  def initialize
+    @settings = get_settings
+    @symbiosis_setting = @settings.symbiosis_setting
+    Flags.add('research-done', 'You finish refreshing your knowledge of the', 'You complete reviewing your knowledge of the')
+    Flags.add('gaf', 'You require some special means of analyzing the mana streams in order to begin research')
+    @time = 300
```

Comment:
What about moving @time into research method as a local variable, since it's only used there? Or not using a variable at all, since it's not defined by any condition, just making it 300 wherever it occurs?

Reply:
Could also set to 300 and it will adjust to the actual time. Perhaps I was thinking, and did not put in yet, an arg for time someone wants to research for but then I thought why would they start and not finish.

Reply:
Replaced with putting in 300.